apps/gr_satellites

```diff
@@ -307,8 +304,6 @@ def main():
     if len(sys.argv) >= 2 and sys.argv[1] in ['--version',
                                               '--list_satellites']:
```

Comment:
We should also remove the `'--list_satellites'` option from here, since it's not useful at all (as you pointed out, the `--list_satellites` option is immediately handled by the action).
The code here should read `if len(sys.argv) >= 2 and sys.argv[1] == '--version'`.

Reply:
That results in useage being shown, the actual parse_args() needs to be reached ?

Reply:
I see. Indeed. I agree this looks good, so let's merge it.